### PR TITLE
feat: improve documentation for reverse selector

### LINKED ISSUE #4209: [ArgoCD][Exclusion] Add a --rejector option to remove object from backup selection.
**Cluster is managed by argoCD and we do not want to backup the argocd-managed objects**

We are using argoCD to manage our cluster. The objects created by argoCD have a specific label ( ie: argocd.argoproj.io/instance: minio ) and we do not want to backup them.

As for today, i did not found any method to reject some of the objects.
This could be a really interesting feature as all the objects managed by helm as well as some kind of labels.

As a sidenote, we would like all objects that are NOT managed by argoCD/helm to be backed up (Thinking about PVs created through a StorageClass )

**Describe the solution you'd like**

The best and  simplest way of doing that would be to create a flag for the backups that looks like the '--selector' flag but instead of calling it '--selector' simply call it '--rejector' that will reject all backups that match the so-called label/value or simply the label key

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "The project would be better with this feature added"
- :-1: for "This feature will not enhance the project in a meaningful way"

### PR DESCRIPTION
**Pull request by JGodin-C2C**
Signed-off-by: Julien Godin <julien.godin@camptocamp.com>

Thank you for contributing to Velero!

# Please add a summary of your change

Add some documentation for the velero selector and explicitly stating that the syntax is kubernetes compatible. 

# Does your change fix a particular issue?

Fixes #4209

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [x] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: site/content/docs/main/resource-filtering.md [hunk 1]
```diff
@@ -88,6 +88,14 @@ Wildcard takes precedence when both a wildcard and specific resource are include
   ```bash
   velero backup create <backup-name> --selector <key>=<value>
   ```
+* Include resources that are not matching the selector
+  ```bash
+  velero backup create <backup-name> --selector <key>!=<value>
+  ```
+
+More information can be found about label selectors here :
```
Comment (a-mccarthy) [inline site/content/docs/main/resource-filtering.md:96]: to make this align with our style guide, please update this link to use markdown and have the link embedded in the text. For example, ```For more information read the [Kuberentes label selector documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)```.

See the velero style guide for more info about link styling, https://velero.io/docs/main/style-guide/#links

**Review by a-mccarthy — CHANGES_REQUESTED** [review]
Thanks for submitting! Just one small nit on link styling

File: site/content/docs/main/resource-filtering.md (hunk 1 above)
Comment (JGodin-C2C) [reply site/content/docs/main/resource-filtering.md]: I used your example and pushed the new version. 
Thanks !

**Review by a-mccarthy — DISMISSED** [review]
LGTM! thanks @JGodin-C2C

**Comment by JGodin-C2C** [comment]
@a-mccarthy Eeeer, it seems that it needs more approval and i cannot add anybody to the review ? 
Do i need to do some action in order to add a reviewer ?

**Comment by a-mccarthy** [comment]
@zubron or @dsu-igeek can you help here? i'm not sure whats missing for approving this PR, thanks!

File: site/content/docs/main/resource-filtering.md [hunk 2]
```diff
@@ -88,6 +88,12 @@ Wildcard takes precedence when both a wildcard and specific resource are include
   ```bash
   velero backup create <backup-name> --selector <key>=<value>
   ```
+* Include resources that are not matching the selector
+  ```bash
+  velero backup create <backup-name> --selector <key>!=<value>
+  ```
+
+For more information read the [Kuberentes label selector documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)
```
Comment (zubron) [inline site/content/docs/main/resource-filtering.md:96]: Can you please fix the spelling of `Kubernetes`? Thanks!

**Review by zubron — CHANGES_REQUESTED** [review]

File: site/content/docs/main/resource-filtering.md (hunk 2 above)
Comment (JGodin-C2C) [reply site/content/docs/main/resource-filtering.md]: Updated

**Review by a-mccarthy — APPROVED** [review]
lgtm! thanks

**Pull request merged**